docs/lang/articles/basic/type.md

```diff
@@ -323,6 +119,9 @@ def foo():
     a = 1
     print(a)  # 1.0
 ```
+In this example, variable `a` is initialized with `float` type. On the next line, the assign statement will cast `1` from `int` type to `float` type implicitly.
+
```

Comment:
In this example, variable `a` is initialized with a `float` type. On the next line, the assignment statement casts `1` from `int` to `float` implicitly.